packages/ansible-lint/ansible-lint.spec

```diff
@@ -17,7 +17,7 @@ BuildRequires:  python%{python3_pkgversion}-setuptools-scm >= 3.5.0
 BuildRequires:  python%{python3_pkgversion}-setuptools_scm_git_archive
 
 %{?python_provide:%python_provide python%{python3_pkgversion}-%{pypi_name}}
-Requires:       ansible >= 2.9
+Requires:       (ansible >= 2.9 or ansible-core)
```

Comment:
This is EL8 only, right? Should this be behind a conditional?

Reply:
No, we only build this package on EL8